data/json/terrain.json

```diff
@@ -2218,7 +2218,8 @@
             "sound_fail": "whump!",
             "ter_set": "t_rock_floor",
             "items": [
-                { "item": "rock", "amount": 7, "minamount": 3 }
+                { "item": "rock", "amount": 7, "minamount": 3 },
+                { "item": "material_limestone", "amount": 25, "minamount": 0, "chance": 80 }
```

Comment:
Thinking minamount 10: cuts down on folks getting 2 limestone for (say) 10 tiles mined out, when it's needed in slightly larger quantities.